plugins/com.google.cloud.tools.eclipse.appengine.deploy.test/src/com/google/cloud/tools/eclipse/appengine/deploy/AppEngineDeployOutputTest.java

```diff
@@ -20,78 +20,67 @@
-            "      \"id\": \"20160429t112518\",\n" +
-            "      \"last_deployed_time\": null,\n" +
-            "      \"project\": \"some-project\",\n" +
-            "      \"service\": \"default\",\n" +
-            "      \"traffic_split\": null,\n" +
-            "      \"version\": null\n" +
-            "    }\n" +
-            "  ]\n" +
-            "}\n";
+        "  \"configs\": [],\n" +
+        "  \"versions\": [\n" +
+        "    {\n" +
+        "      \"id\": \"20160429t112518\",\n" +
+        "      \"last_deployed_time\": null,\n" +
+        "      \"project\": \"some-project\",\n" +
+        "      \"service\": \"default\",\n" +
+        "      \"traffic_split\": null,\n" +
+        "      \"version\": null\n" +
+        "    }\n" +
+        "  ]\n" +
+        "}\n";
 
     AppEngineDeployOutput deployOutput =
         AppEngineDeployOutput.parse(jsonOutput);
     Assert.assertEquals("20160429t112518", deployOutput.getVersion());
     Assert.assertEquals("default", deployOutput.getService());
   }
 
-  @Test
+  @Test(expected = JsonParseException.class)
   public void testDeployOutputJsonParsingTwoVersions() {
     String jsonOutput =
         "{\n" +
-            "  \"configs\": [],\n" +
-            "  \"versions\": [\n" +
-            "    {\n" +
-            "      \"id\": \"20160429t112518\",\n" +
-            "      \"last_deployed_time\": null,\n" +
-            "      \"project\": \"some-project\",\n" +
-            "      \"service\": \"default\",\n" +
-            "      \"traffic_split\": null,\n" +
-            "      \"version\": null\n" +
-            "    },\n" +
-            "    {\n" +
-            "      \"id\": \"20160429t112518\",\n" +
-            "      \"last_deployed_time\": null,\n" +
-            "      \"project\": \"some-project\",\n" +
-            "      \"service\": \"default\",\n" +
-            "      \"traffic_split\": null,\n" +
-            "      \"version\": null\n" +
-            "    }\n" +
-            "  ]\n" +
-            "}\n";
+        "  \"configs\": [],\n" +
+        "  \"versions\": [\n" +
+        "    {\n" +
+        "      \"id\": \"20160429t112518\",\n" +
+        "      \"last_deployed_time\": null,\n" +
+        "      \"project\": \"some-project\",\n" +
+        "      \"service\": \"default\",\n" +
+        "      \"traffic_split\": null,\n" +
+        "      \"version\": null\n" +
+        "    },\n" +
+        "    {\n" +
+        "      \"id\": \"20160429t112518\",\n" +
+        "      \"last_deployed_time\": null,\n" +
+        "      \"project\": \"some-project\",\n" +
+        "      \"service\": \"default\",\n" +
+        "      \"traffic_split\": null,\n" +
+        "      \"version\": null\n" +
+        "    }\n" +
+        "  ]\n" +
+        "}\n";
 
-    try {
-      AppEngineDeployOutput.parse(jsonOutput);
-      Assert.fail("Failure to throw exception when parsing deploy output with more that one version entry");
-    } catch (JsonParseException e) {
```

Comment:
In these cases it can be useful to assert something about the exception such as the message